ibis/expr/types/geospatial.py

```diff
@@ -455,6 +973,10 @@ def start_point(self) -> PointValue:
         -------
         PointValue
             Start point
+
+        Example
+        -------
+        ?? this only applies to lines and no lines in t.geom
```

Comment:
This method is specific too `LineStrings`, and we don't have a way to create one from the table to get an example

Reply:
A literal seems fine here, would that work?